features/git-town-ship/current_branch/on_feature_branch/on_parent_branch.feature

```diff
@@ -42,11 +42,11 @@ Feature: git town-ship: shipping a parent branch
 
 
   Scenario: undo
-    When I run `git town-ship --undo`
+    When I run `gt ship --undo`
     Then it runs the commands
       | BRANCH         | COMMAND                                                      |
       | main           | git branch parent-feature <%= sha 'parent feature commit' %> |
-      |                | git revert <%= sha 'parent feature done' %>                  |
+      |                | git revert HEAD                                              |
```

Comment:
Unless I am misunderstanding you, we should be able to determine the SHA of the commit we want to revert to __before__ we run the squash-merge step, no?